src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
```

Comment:
Use the method shorthand syntax for defining methods. Instead of `add: function(number) { ... }`, you can simply use `add(number) { ... }`.